feature-libs/product-configurator/rulebased/components/overview-notification-banner/configurator-overview-notification-banner.component.ts

```diff
@@ -29,16 +29,7 @@ export class ConfiguratorOverviewNotificationBannerComponent {
     switchMap((routerData) =>
       this.configuratorCommonsService.getConfiguration(routerData.owner)
     ),
-    map((configuration) => {
-      if (configuration.overview?.totalNumberOfIssues) {
-        return configuration.overview.totalNumberOfIssues;
-      } else if (configuration.totalNumberOfIssues) {
-        return configuration.totalNumberOfIssues;
-      } else {
-        //TODO fix before merging to develop. We can never reach that line
-        return 0;
-      }
-    })
+    map((configuration) => configuration?.overview?.totalNumberOfIssues)
```

Comment:
Check if this change is possible due to backward compatibility, because this component is already in VC configurator.
This _fix_ is needed because otherwise the overview page will show, for a brief moment, an incorrect number of issues for CPQ configurations.

Reply:
maybe It's required for CPS?

Reply:
@schuppell : have a look to this file in develop:
https://github.com/SAP/spartacus/blob/a071ba7f89e0d0720e436c1577194f7a1330fe1d/feature-libs/product-configurator/rulebased/components/overview-notification-banner/configurator-overview-notification-banner.component.ts#L38
seems like CPS is reading this from configuration (Not overview). I guess this change would break this functionality.
So we another solution here.

Reply:
maybe just delete the else with the ToDo and change the test, so it waits until the link contains the expected number. The fact that it shortly display 6 instead of 8 issues, is related to the way we model the error messages in CPQ and that they are not populated when we fire a request to the same tab.
We should not bother to much with this.

Reply:
ok, changed back

Reply:
@schuppell did you push?